bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/transactions/TransactionResourceManager.java

```diff
@@ -126,7 +128,7 @@ public static TransactionResourceManager getInstance() {
      *
      */
     private void setLogProperties() {
-        final Path projectRoot = findProjectRoot(Paths.get(System.getProperty("user.dir")));
+        final Path projectRoot = Paths.get(System.getProperty(USER_DIR));
         if (projectRoot != null) {
             String logDir = getTransactionLogDirectory();
             String logPath = projectRoot.toAbsolutePath().toString() + File.separatorChar + logDir;
```

Comment:
Better check !isAbsolute() and append else set the abosolute path

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/27960/commits/720dd6d64bc0f6387f7ea1102ab8e848fe134718